
syntax = "proto3";

package biohub;

import "google/protobuf/timestamp.proto";

// Example entity with a timestamp
message ExampleEntity {
    string id = 1; // Unique identifier
    string name = 2; // Example name
    google.protobuf.Timestamp created_at = 3; // Timestamp for creation
    google.protobuf.Timestamp updated_at = 4; // Timestamp for last update
}
